### ISSUE TITLE
paddle.Model.fit在eval阶段不打印传入的metrics结果，最优结果的保存似乎也没有

### ISSUE THREAD

**Issue by bbbxixixixi** [PaddlePaddle/PaddleNLP#170]
用SimNet做了个语义判断的程序
model = ppnlp.models.SimNet(
        network='gru',
        emb_dim=256,
        vocab_size=len(vocab),
        num_classes=3)
model = paddle.Model(model)
optimizer = paddle.optimizer.AdamW(
        parameters=model.parameters(), learning_rate=0.001)
criterion = paddle.nn.CrossEntropyLoss()
metric = paddle.metric.Accuracy()
model.prepare(optimizer, criterion, metric)
model.fit(
        train_loader,
        evl_loader,
        epochs=20,
        save_dir='pretrained_model' )
发现在eval阶段只是输出loss等信息，并不会打印出整个eval data的评价指标
然后在最后结束时只存了个final model，paddlex会存一个best model，paddlenlp现在会根据eval阶段的最优解指标存一个模型吗？

**Comment by Steffy-zxf** [comment]
你好！`paddle.Model.fit`在eval阶段会打印eval data的评价指标。并且打印的指标是从一个累积的数值。同时`paddle.Model.fit`可以指定`save_freq`参数，间隔epoch数之后保存模型参数，参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/hapi/model/Model_cn.html#model

simnet训练示例代码参考：https://github.com/PaddlePaddle/PaddleNLP/tree/release/2.0-rc/examples/text_matching/simnet
